app/src/main/java/org/fossasia/openevent/app/core/faq/list/FaqListPresenter.java

```diff
@@ -75,4 +81,43 @@ private void listenChanges() {
     public List<Faq> getFaqs() {
         return faqs;
     }
+
+    public void deleteFaqWithId(long id, boolean forceReload) {
+        faqRepository
+            .deleteFaq(id)
+            .compose(disposeCompletable(getDisposable()))
+            .compose(progressiveErroneousCompletable(getView()))
+            .subscribe(() -> {
+                getView().showMessage("FAQ Deleted. Refreshing Items");
+                loadFaqs(forceReload);
+            }, Logger::logError);
+    }
+
+    public void deleteSelectedFaq() {
+        deleteFaqWithId(faqId, true);
+    }
+
+    public void selectSpecificFaq(List<Faq> faqs, Long faqId, boolean selection) {
```

Comment:
What does this do?

Reply:
When a list item is selected and then the layout is swipe refreshed then I need to deselect all the list items. Also when an item is selected and then other item is long pressed then I need to deselect all the list items other than that. The faqId method parameter contains the id of the list item which needs to be selected. This method address to the above two cases under if and else respectively.

Reply:
On rotation, the ticket Fragment, FAQ Fragment, Attendee Fragment, Event Fragment all gets destroyed and user lands on Dashboard. So I think persisting the fragment on screen rotation should be ensured first as a separate issue. What do you say?

Reply:
Yes, alright